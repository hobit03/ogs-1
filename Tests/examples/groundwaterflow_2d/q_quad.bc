GeoSys-BC: Boundary Conditions ------------------------------------------------
#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE
  HEAD
 $GEO_TYPE
  POLYLINE BC
 $DIS_TYPE
  CONSTANT 0
#STOP

#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE
  HEAD
 $GEO_TYPE
  POLYLINE Channel
 $DIS_TYPE
  CONSTANT 10
#STOP
