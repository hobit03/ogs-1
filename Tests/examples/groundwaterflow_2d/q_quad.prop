<?xml version="1.0"?>
<coupling>
	<P algorithm="Serial" convergence="FemFunctionConvergenceCheck" max_itr="1" epsilon="1e-4">
		<out>HEAD</out>
		<out>VELOCITY</out>
		<problems>
			<M name="GROUNDWATER_FLOW">
				<out>HEAD</out>
			</M>
			<M name="HEAD_TO_ELEMENT_VELOCITY">
				<in>HEAD</in>
				<out>VELOCITY</out>
			</M>
		</problems>
	</P>
</coupling>

