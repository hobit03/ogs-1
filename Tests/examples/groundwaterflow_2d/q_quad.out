GeoSys-OUT: Output ------------------------------------------------
#OUTPUT
 $NOD_VALUES
  HEAD
 $ELE_VALUES
  VELOCITY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP

 $PCS_TYPE
  GROUNDWATER_FLOW

#OUTPUT
 $PCS_TYPE
  ELEMENT_VELOCITY
 $ELE_VALUES
  VELOCITY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP

#OUTPUT
 $NOD_VALUES
  HEAD
 $GEO_TYPE
  POINT POINT4
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
 STEPS: 1
